pubspec.yaml

```diff
@@ -38,10 +38,16 @@ dev_dependencies:
   dependency_validator: ^1.2.2
   mockito: ^4.0.0
   over_react_test: ^2.5.2
+  redux_dev_tools: ^0.4.0
+  redux_remote_devtools: ^0.0.7
   test: ^1.0.0
 
 dependency_overrides:
   react:
     git:
       url: https://github.com/cleandart/react-dart.git
       ref: 5.1.0-wip
+  socketcluster_client:
+    git:
+      url: https://github.com/kealjones-wk/socketcluster_client.git
+      ref: auto-platform-selection
```

Comment:
If we can't get this merged into `socketcluster_client`, we may need to move the redux demo into the example dir, and make the example have its own pubspec, etc - so we can place the dependency override there.

Reply:
👍 That makes sense, I wasn't sure the best way to handle having the PR in progress. Since `3.1.0-wip` isn't released yet, would we keep this here until we look at releasing (or it gets merged) and evaluate then?

Reply:
🎉 🎉 🎉 It merged!! 🎉 🎉 🎉